uiowa/uiowa
# Programmed Classrooms View

Resolves #6131 




# How to test
-  Code review 
  - Room list block and Programmed list block now use the same code
- `ddev blt ds --site=classrooms.uiowa.edu`
- Add a new page and add the "Room List" and "Programmed List" blocks
  - Test out programmed list filters and verify that these fields display:
    -  Building (Should always display)
    -  Room name (only displays for rooms that have it)
    -  Max Occupancy (Should always display)
    -  Responsible Unit (Should always display)
    -  Seating Type (only displays for those that have it)
 - Verify that the "Room list" displays only Image, Max Occupancy and Room type only. 
 - Check helper function in uiowa_core that allowed removal of a few field template files. 
   - Good naming and usage?
-  On a room page, check that that the request info buttons only display if the room is not a "Programmed Classroom"
